### PR TITLE
Add highlights for the week ending 2022-05-27

### PR DESCRIPTION
**Pull request by weatherhog**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Comment by gianfranco-l** [comment]
as I'll be AFK for the next 2 days, I left a note for @giantswarm/team-honeybadger to provide with team contributions. In case you don't hear from the team on Friday morning, please feel free to ping @MarcelMue and/or @piontec

File: src/content/changes/highlights/2022-05-27.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## General
+
+## Apps
+
+## User interfaces
+
+## Documentation
```
Comment (marians) [inline src/content/changes/highlights/2022-05-27.md:15]:
```suggestion
## User interfaces

In our web UI's new function to inspect permission, we now also show `kubectl` commands to gather similar information from the Management API using a CLI.

## Documentation
```

File: src/content/changes/highlights/2022-05-27.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: Highlights for the week ending May 27, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## General
+
+## Apps
+
```
Comment (MarcelMue) [inline src/content/changes/highlights/2022-05-27.md:12]:
```suggestion
[flux-app](https://github.com/giantswarm/flux-app/) version [v0.11.0](https://github.com/giantswarm/flux-app/blob/master/CHANGELOG.md#0110---2022-05-26) updates to upstream version [v0.30.2](https://github.com/fluxcd/flux2/releases/tag/v0.30.2).
```

Comment (MarcelMue) [reply src/content/changes/highlights/2022-05-27.md:12]: Does this work @piontec @kubasobon ? Do we need some added disclaimer?

**Review by marians — APPROVED** [review]

**Pull request merged**